onecore_maintenance_extension/views/maintenance_views.xml

```diff
@@ -226,11 +226,26 @@
                     <field name="space_caption" />
                     <field name="create_date" />
                     <field name="master_key" />
+                    <field name="new_mimer_notification" />
                 </xpath>
                 <!-- Remove existing fields regarding equipment -->
                 <xpath expr="//div[@class='o_kanban_record_body']/span[1]" position="replace" />
                 <xpath expr="//div[@class='o_kanban_record_body']/span[2]" position="replace" />
                 <xpath expr="//div[@class='o_kanban_record_bottom']" position="replace" />
+                <xpath expr="//div[@class='o_kanban_record_top']" position="replace">
+                  <div class="o_kanban_record_top">
+                    <div class="w-100 d-flex justify-content-between align-items-center">
```

Comment:
Use the class `.mimer-badge` in `onecore_maintenance_extension/static/src/scss/mimer_styles.scss`

Reply:
🤯